guide/onboarding/protecting-a-wallet.md

```diff
@@ -0,0 +1,223 @@
+
+Unfortunately, biometrics are not supported on all mobile devices. Hardcore security users may be wary about giving up their biometric information.
+
+</div>
+
+{% include /tip/open.html label="Tip: Be Transparent" icon="info" color="blue" %}
+
+Bitcoin wallet users may feel hesitant about the privacy implications of your product storing touch/face ID information. Be sure your app thoroughly relays how this information is stored.
+
+{% include /tip/close.html %}
+
+### PIN
+
+A PIN is one of the most common and traditional methods for securing a mobile application. This setup requires that users create a 4-6 digit PIN that must be entered when opening their wallet application, or when viewing sensitive information such as their recovery phrase.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/choose-pin.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/choose-pin@2x.png"
+   alt-text = "Choosing a pin screen"
+   width = 250
+   height = 541
+   class = "shadow"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/enter-pin.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/enter-pin@2x.png"
+   alt-text = "Enter pin screen"
+   width = 250
+   height = 541
+   class = "shadow"
+%}
+
+</div>
+
+The PIN-based approaches can be supported by all devices, and therefore is more accessible. A PIN can also be used to encrypt data within a wallet, which makes any locally stored information on the device inaccessible. Additionally, some users prefer this type of security measure because they use the same or similiar PINs on multiple applications/devices, making it easy to remember.
+
+However, users with poor security practices might create easy-to-guess PINs (such as 1234), which would decrease the level of security. This design also relies on users to remember their PIN correctly.
+
+{% include tip/open.html color="green" icon="check" label="Do: Educate on Good Security Practices" %}
+
+Within your product upon PIN creation, tell your users what a good, secure PIN is (i.e. not their birthday). Recommend against (or even make it impossible) to create a PIN such as 1234, 1111, etc.
+
+{% include tip/close.html %}
+
+
+### Hiding Sensitive Information
+
+Imagine this scenario. You are in a public place, and you need to make a payment using your bitcoin wallet. You open your wallet on your phone, but you don’t feel comfortable having your address and balance information clearly visible to strangers who may be looking over your shoulder, persons lurking or video surveillance . Hence by giving users the ability to hide sensitive information in their wallet if desired, they gain an added sense of physical privacy and security [^3]when using the app in public.
+
+### What information is considered sensitive ?
```

Comment:
We already talk about what information is sensitive, private in the intro paragraph, perhaps merge in the information?

> Bitcoin wallets contain sensitive information such as recovery phrases, balances, transaction histories, and more. Therefore, it is crucial to add extra layers of security to keep private data out of bad actors' hands. Additional protection may also be necessary for users who share devices, such as parents with children.

Not a blocker, but could be improved to avoid repetition.